dev.fredol.open-tv.yaml

```diff
@@ -0,0 +1,1152 @@
+id: dev.fredol.open-tv
+
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+
+command: open_tv
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --filesystem=host:ro
```

Comment:
why is host needed ?

Reply:
the app can't render the right fonts otherwise, I wasn't sure what to add to fix it. Also might be better for a clueless user loading a big file outside of the flatpak allowed paths. Some of these m3u files can be 500 mb big.